### ISSUE TITLE
[Bug]: Auto-Hammer not crushing deepslate or cobbled-deepslate 

### ISSUE THREAD

**Issue by channeyy123** [FTBTeam/FTB-Modpack-Issues#1602]
### Modpack

FTB Stoneblock 3 

### Modpack version

1.1

### Describe the bug

Cannot automate the creation of crushed deepslate in the auto-hammer. The auto-hammer does not process the input of cobbled deepslate or deepslate. 

REI shows auto-hammer as a crushing tool for crushing deepslate 

### Steps to reproduce

1. Place either deepslate or cobbled deepslate in a chest
2.  place auto-hammer next to chest with blue input facing the chest 
3. place another chest on the red output of the auto-hammer
4.  auto-hammer does not process 

### Expected behaviour

Auto-hammer should process the deepslate and produce crushed deepslate

**Comment by AWDoBaun** [comment]
I'd love to say  that there is nothing in the pack that actually says that deepslate requires a diamond hammer or auto-hammer, as such there is a failing in the recipes as presented in REI.
![image](https://user-images.githubusercontent.com/12954977/200213544-8b646dd9-efbd-4bd5-aabe-3ebd23a2a5f9.png)
It is apparent on screen, if you know what to look for, when you wave an actual hammer at a block of deepslate. For example an iron hammer in hand looking at deepslate shows the very tiny red x on the pickaxe to indicate the tool is inadequate.:
![image](https://user-images.githubusercontent.com/12954977/200213651-8508139b-24fb-4dee-a916-d57e1478d010.png)

But the fact that the recipes in REI seem to declare deepslate can be hammered by stone hammers or by an iron autohammer is kind of a bug in how that recipe is presented. And it is the reason why this "bug" was reported. I came to report the same. And it was only while writing it up that I took the picture with the iron hammer in hand and realized it might require a higher level tool.
